doc/help/elektra-algorithm.md

```diff
@@ -304,7 +304,7 @@ The user passes the configuration using the `KeySet` `returned`.  The key
 set will not be changed by `kdbSet()`.	The `parentKey` provides a way
 to limit which part of the configuration is written out.  For example,
 the `parentKey` `user/sw/org/app/#0/current` will induce `kdbSet()` to
-only modify the key databases below `user/sw/apps/myapp` even
+only modify the key databases below `user/sw/apps/app` even
```

Comment:
> Maybe "org" instead of "apps" would be better? (again the problem already existed earlier)

I totally overlooked that mistake 😳. The new version of the pull request contains the correct key.